### PR TITLE
Add clarification around video tile grid documentation

### PR DESCRIPTION
**Pull request by michhyun1**
**Issue #:** 
https://github.com/aws/amazon-chime-sdk-component-library-react/issues/163

**Description of changes:**
Update some documentation

**Testing**
1. Have you successfully run `npm run build:release` locally? yes

2. How did you test these changes? n/a

3. If you made changes to the component library, have you provided corresponding documentation changes?
Yes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/components/sdk/VideoTileGrid/VideoTileGrid.stories.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { VideoTileGrid } from './';
 
 # VideoTileGrid
 
-The `VideoTileGrid` component renders all meeting session video tiles in a responsive grid layout. This includes the local tile, remote tiles, and content share tile.
+The `VideoTileGrid` component renders all meeting session video tiles in a responsive grid layout. This includes the local tile, remote tiles, and content share tile. By default, a user joins without video, so in order to see a video tile appear, you must first turn on a video tile.
```
Comment (bradmanAWS) [inline src/components/sdk/VideoTileGrid/VideoTileGrid.stories.mdx:8]: I think builders were expecting the video grid to appear. Maybe we could add something like: in order for the VideoTileGrid to render, at least one video tile must be shared.

Comment (michhyun1) [reply src/components/sdk/VideoTileGrid/VideoTileGrid.stories.mdx]: cool, updated

File: src/components/sdk/VideoTileGrid/VideoTileGrid.stories.mdx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { VideoTileGrid } from './';
 
 # VideoTileGrid
 
-The `VideoTileGrid` component renders all meeting session video tiles in a responsive grid layout. This includes the local tile, remote tiles, and content share tile.
+The `VideoTileGrid` component renders all meeting session video tiles in a responsive grid layout. This includes the local tile, remote tiles, and content share tile. By default a user joins without video, so in order to see the VideoTileGrid there must be at least one video tile being shared.
```
Comment (p-foucht) [inline src/components/sdk/VideoTileGrid/VideoTileGrid.stories.mdx:8]: Can we add something about starting video?

`To start sharing a video, see the "VideoButton or w/e" component`

Comment (michhyun1) [reply src/components/sdk/VideoTileGrid/VideoTileGrid.stories.mdx:8]: Got it, will update.

**Pull request merged**